tests/check/check.go

```diff
@@ -73,36 +67,6 @@ func testCheck(t *testing.T, client ClientInterface) {
 	})
 }
 
-func testBadAuthModelID(t *testing.T, client ClientInterface) {
```

Comment:
Moved to https://github.com/openfga/openfga/pull/1266/files#diff-13b105826200b5ebee7b7a8308e113e492ce11f13ad0aaf5652fa78fe0a4d416R1012